index.html

```diff
@@ -1812,31 +1791,29 @@ <h2>
             </p>
             <p data-tests=
             "change-shipping-option-select-last-manual.https.html">
-              If an item in the sequence has the <a data-lt=
-              "PaymentShippingOption.selected">selected</a> member set to true,
-              then this is the shipping option that will be used by default and
-              <a data-lt="PaymentRequest.shippingOption">shippingOption</a>
-              will be set to the <a data-lt="PaymentShippingOption.id">id</a>
-              of this option without running the <a>shipping option changed
-              algorithm</a>. If more than one item in the sequence has
-              <a data-lt="PaymentShippingOption.selected">selected</a> set to
+              If an item in the sequence has the
+              {{PaymentShippingOption/selected}} member set to true, then this
+              is the shipping option that will be used by default and
+              {{PaymentRequest/shippingOption}} will be set to the
+              {{PaymentShippingOption/id}} of this option without running the
+              <a>shipping option changed algorithm</a>. If more than one item
+              in the sequence has {{PaymentShippingOption/selected}} set to
               true, then the <a>user agent</a> selects the last one in the
               sequence.
             </p>
             <p>
               The <a>shippingOptions</a> member is only used if the
               {{PaymentRequest}} was constructed with <a>PaymentOptions</a> and
```

Comment:
Might want to check if there are any other strays: 
```suggestion
              {{PaymentRequest}} was constructed with {{PaymentOptions}} and
```